i18n/fr/docusaurus-plugin-content-docs/version-22.04/metrology/grafana.md

```diff
@@ -27,6 +29,45 @@ Les données de performance sont disponibles, mais pas des données telles que l
 
 Le plugin Centreon pour Grafana est disponible sur [la page de téléchargement de Centreon](https://download.centreon.com/).
 
+## Comment mettre en place le plugin?
+
+Vous pouvez utiliser la data source Centreon pour Grafana en tant que plugin non signé avec votre propre installation Grafana. (Ce n'est pas possible avec une instance Cloud.)
+
+Pour utiliser la data source Centreon en tant que plugin non signé :
+
+1. Assurez-vous que vous disposez des droits d'administration sur la machine sur laquelle Grafana s'exécute.
+
+2. Téléchargez le fichier zippé contenant la data source depuis la [page de téléchargement de Centreon](https://download.centreon.com/). La data source se situe dans la sous-section **Grafana** de la section **Custom platform**.
+
+3. Extrayez la data source et placez-la dans le répertoire **plugins** de votre installation Grafana. (L'emplacement de ce répertoire est défini dans la variable **plugins** de votre fichier **grafana/conf/defaults.ini**.)
+
+4. Si vous n'avez pas de fichier init personnalisé, faites une copie du fichier **default.ini** et renommez-le **custom.ini**.
+
```

Comment:
C'est un peu un détail pcq on  parle probablement à des gens qui savent ce qu'il font mais ce n'est pas cohérent. Si à l'étape 4, l'utlisateur a déjà un fichier custom.in alors à l'étape 3 c'est dans ce fichier qu'il faut aller regarder et pas dans le default.ini . (custom.ini va surcharger default.ini)
Donc, il faut sans doute inverser 3 et 4 et dire que le chemin vers plugins doit être vérifié dans le custom.ini.